src/drivers/BoostDriver.cpp

```diff
@@ -60,8 +60,8 @@ class CukeBoostLogInterceptor : public ::boost::unit_test::unit_test_log_formatt
     void log_exception_finish( std::ostream& ) {};
 
     void log_entry_start( std::ostream&, log_entry_data const&, log_entry_types /*let*/) {};
-    void log_entry_value( std::ostream&, const_string /*value*/);
-    void log_entry_value( std::ostream&, lazy_ostream const& /*value*/) {};
+    void log_entry_value( std::ostream&, const_string value);
+    void log_entry_value( std::ostream&, lazy_ostream const& value);
```

Comment:
Oh, nevermind, I didn't notice that you changed the empty inline function (`{}`) to just a declaration with the function's definition elsewhere.

As for warnings: you should compile with `-Wall -Wextra` to see any. I think we should probably consider building with that one plus `-Werror` by default on our CI systems at least.

The old code (with empty inline function definition) without commented out parameter produces this with `-DCMAKE_CXX_FLAGS='-Wall -Wextra -Werror':
```
../src/drivers/BoostDriver.cpp:64:62: error: unused parameter 'value' [-Werror,-Wunused-parameter]                                                                 void log_entry_value( std::ostream&, lazy_ostream const& value) {};                                                               
                                                             ^
```